app/shared/actions/wallets.js

```diff
@@ -0,0 +1,67 @@
+import { find, forEach } from 'lodash';
+
+import * as types from './types';
+import { getAccount } from './accounts';
+import { setSettings } from './settings';
+import { encrypt } from './wallet';
+
+export function importWallet(account, key = false, password = false, mode = 'hot') {
```

Comment:
I like the way you broke up wallet actions methods into wallet and wallets methods. It's really intuitive 😄